src/settings-ui/Settings.UI/Controls/CheckBoxWithDescriptionControl.cs

```diff
@@ -40,12 +40,12 @@ private void CheckBoxSubTextControl_Loaded(object sender, RoutedEventArgs e)
             // Add text box only if the description is not empty. Required for additional plugin options.
             if (!string.IsNullOrWhiteSpace(Description))
             {
-                panel.Children.Add(new TextBlock() { Margin = new Thickness(0, 10, 0, 0), Text = Header });
+                panel.Children.Add(new TextBlock() { Margin = new Thickness(0, 10, 0, 0), Text = Header, TextWrapping = TextWrapping.WrapWholeWords });
                 panel.Children.Add(new IsEnabledTextBlock() { Style = (Style)App.Current.Resources["SecondaryIsEnabledTextBlockStyle"], Text = Description });
             }
             else
             {
-                panel.Children.Add(new TextBlock() { Margin = new Thickness(0, 0, 0, 0), Text = Header });
+                panel.Children.Add(new TextBlock() { Margin = new Thickness(0, 0, 0, 0), Text = Header, TextWrapping = TextWrapping.WrapWholeWords });
```

Comment:
It should be possible to allign margins back to the same value now?

Reply:
I wonder why these `two Add(new TextBlock` bits are called twice in the first place - both in the `if` and the `else` statements. Why isn't this line _before_ the `if` statement?

Reply:
This was a workaround to limit the problems with the margins. I have added it in the PR that implemented optional description property for additional plugin options.